### DISCUSSION TITLE
Pair Extraordinaire x4

### DISCUSSION THREAD

**Discussion by Stratis-Dermanoutsos** [Schweinepriester/github-profile-achievements#643] [Q&A] [answered]
I have about 50+ merged co-authored PRs and I haven't received the gold achievement.

Any ideas what's wrong?

**Comment by Schweinepriester** [comment, marked as answer]
Unfortunately it would appear that the current counts are not entirely precise, see <https://github.com/Schweinepriester/github-profile-achievements/pull/247>…

**Reply by Stratis-Dermanoutsos** [reply]
This seems to be the case indeed.
